cli/src/main/java/org/infinispan/cli/commands/rest/Connector.java

```diff
@@ -0,0 +1,216 @@
+      protected boolean help;
+
+      @Override
+      public boolean isHelp() {
+         return help;
+      }
+
+      @Override
+      protected CompletionStage<RestResponse> exec(ContextAwareCommandInvocation invocation, RestClient client, Resource resource) {
+         return client.server().connectorNames();
+      }
+   }
+
+   @CommandDefinition(name = "describe", description = "Describes a connector", activator = ConnectionActivator.class)
+   public static class Describe extends RestCliCommand {
+
+      @Argument(required = true, completer = ConnectorCompleter.class)
+      String name;
+
+      @Option(shortName = 'h', hasValue = false, overrideRequired = true)
+      protected boolean help;
+
+      @Override
+      public boolean isHelp() {
+         return help;
+      }
+
+      @Override
+      protected CompletionStage<RestResponse> exec(ContextAwareCommandInvocation invocation, RestClient client, Resource resource) {
+         return client.server().connector(name);
+      }
+   }
+
+   @CommandDefinition(name = "start", description = "Starts a connector", activator = ConnectionActivator.class)
+   public static class Start extends RestCliCommand {
+
+      @Argument(required = true, completer = ConnectorCompleter.class)
+      String name;
+
+      @Option(shortName = 'h', hasValue = false, overrideRequired = true)
+      protected boolean help;
+
+      @Override
+      public boolean isHelp() {
+         return help;
+      }
+
+      @Override
+      protected CompletionStage<RestResponse> exec(ContextAwareCommandInvocation invocation, RestClient client, Resource resource) {
+         return client.server().connectorStart(name);
+      }
+   }
+
+   @CommandDefinition(name = "stop", description = "Stops a connector", activator = ConnectionActivator.class)
+   public static class Stop extends RestCliCommand {
+
+      @Argument(required = true, completer = ConnectorCompleter.class)
+      String name;
+
+      @Option(shortName = 'h', hasValue = false, overrideRequired = true)
+      protected boolean help;
+
+      @Override
+      public boolean isHelp() {
+         return help;
+      }
+
+      @Override
+      protected CompletionStage<RestResponse> exec(ContextAwareCommandInvocation invocation, RestClient client, Resource resource) {
+         return client.server().connectorStop(name);
+      }
+   }
+
+   @GroupCommandDefinition(name = "ipfilter", description = "Manages connector IP filters", activator = ConnectionActivator.class, groupCommands = {IpFilter.Ls.class, IpFilter.Clear.class, IpFilter.Set.class})
+   public static class IpFilter extends CliCommand {
+
+      @Option(shortName = 'h', hasValue = false, overrideRequired = true)
+      protected boolean help;
+
+      @Override
+      protected boolean isHelp() {
+         return false;
+      }
+
+      @Override
+      protected CommandResult exec(ContextAwareCommandInvocation invocation) throws CommandException {
+         return CommandResult.SUCCESS;
```

Comment:
shouldn't be failure? or maybe remove `ls` command and list the ip filter rules here.